tests/ProxyManagerTest/ProxyGenerator/Util/PropertiesTest.php

```diff
@@ -187,4 +187,45 @@ public function testGetInstanceProperties()
 
         $this->assertCount(9, $properties->getInstanceProperties());
     }
+
+    /**
+     * @dataProvider propertiesToSkipFixture
+     *
+     * @param string $propertyName with property name
+     * @param string $methodToGetProperties name of method to acces the property
+     */
+    public function testSkipPropertiesByFiltering($propertyName, $methodToGetProperties)
+    {
+        $properties = Properties::fromReflectionClass(
+            new ReflectionClass(ClassWithMixedProperties::class)
+        );
+
+        $this->assertArrayHasKey($propertyName, $properties->$methodToGetProperties());
```

Comment:
Can this simply be re-designed to use `getInstanceProperties()`? I don't like the dynamic method call here :-)